docs/topics/multiplatform/multiplatform-expect-actual.md

```diff
@@ -0,0 +1,549 @@
+the `java.time.LocalDate` class to implement the second. However, there's no way to add the `actual` keyword directly to
+these types.
+
+Instead, you can use [type aliases](type-aliases.md) to connect the expected declarations and the platform-specific
+types:
+
+```kotlin
+actual typealias Month = java.time.Month
+actual typealias MyDate = java.time.LocalDate
+```
+
+The `typealias` declaration should be defined in the same package as the expected declaration, while the referred class
+should be defined elsewhere.
+
+> Since the `LocalDate` type uses the `Month` enum, you need to declare both of them as expected classes in common code.
+>
+{type="note"}
+
+<!-- See [Using platform-specific APIs](multiplatform-connect-to-apis.md#actualizing-an-interface-or-a-class-with-an-existing-platform-class-using-typealiases)
+for an Android-specific example of this pattern. -->
+
+### Widened visibility in actual declarations
+
+You can make actual implementations more visible than the corresponding expected declaration. This is useful if you
+don't want to expose your API as public for common clients.
+
+Currently, the Kotlin compiler issues an error in case of visibility changes. You can suppress this error by
+applying `@Suppress("ACTUAL_WITHOUT_EXPECT")` to the actual type alias declaration. This limitation will be lifted
+starting with Kotlin 2.0.
+
+For example, if you declare the following expected declaration in the common source set:
+
+```kotlin
+internal expect class Messenger {
+    fun sendMessage(message: String)
+}
+```
+
+Then, you can use the following actual implementation in a platform-specific source set as well:
+
+```kotlin
+@Suppress("ACTUAL_WITHOUT_EXPECT")
+public actual typealias Messenger = MyMessenger
+```
+
+Here, an internal expected class has an actual implementation with an existing public `MyMessenger` using type aliases.
+
+### Additional enumeration entries on actualization
+
+When an enumeration is declared with `expect` in the common source set, each platform module should have a
+corresponding `actual` declaration. These declarations must contain the same enum constants, but they can have
+additional constants too.
+
+This is very useful when you actualize an expected enum with an existing platform enum. For example, consider the
+following enumeration in the common source set:
+
+```kotlin
+// In the commonMain source set:
+expect enum class Department { IT, HR, Sales }
+```
+
+When you provide an actual declaration for `Department` in platform source sets, you can add extra constants:
+
+```kotlin
+// In the jvmMain source set:
+actual enum class Department { IT, HR, Sales, Legal }
+```
+
+```kotlin
+// In the nativeMain source set:
+actual enum class Department { IT, HR, Sales, Marketing }
+```
+
+However, matching `Department` in common code can never be exhaustive now. Therefore, the compiler requires you
```

Comment:
I don't understand this sentence. Does it mean that all possible cases need to be covered?